dev/release/post-08-docs.sh

```diff
@@ -84,6 +84,12 @@ if [ "$is_major_release" = "yes" ] ; then
   previous_series=${previous_version%.*}
   mv docs_temp docs/${previous_series}
 fi
+# Update DOCUMENTATION_OPTIONS.theme_switcher_version_match and
+# DOCUMENTATION_OPTIONS.show_version_warning_banner
+cd docs/${previous_version}
+find ./ -type f -exec sed -i "s/DOCUMENTATION_OPTIONS.theme_switcher_version_match = '';/DOCUMENTATION_OPTIONS.theme_switcher_version_match = '${previous_version}';/g" {} \;
```

Comment:
Oh, didn't know that. Thank you for pointing it out!

If I understand correctly, all the `.html` files will first be copied (backed-up) to `*.bak` files and then the original files (not copied `*.bak` files) will be corrected if needed? And after substituting we delete the backed up files?

Hope this will not cause an issue in the time to finish the task (lots of html files!)?